### DISCUSSION TITLE
Additional software in BIRT All in one?

### DISCUSSION THREAD

**Discussion by rene-lange** [eclipse-birt/birt#1530] [Q&A]
Is the BIRT All in one version of Eclipse meant to be extended like the "original" version of Eclipse which has the Eclipse marketplace?
I am new and trying to understand how to get plugins for Git and Maven installed.
I have trouble adding the right Software Sites.

Thanks for any support.

**Comment by merks** [comment]
I think in principle it should be possible to install anything you'd normally installed into any other IDE.    

It's also possible use any of these IDEs:

https://www.eclipse.org/downloads/packages/

And then to install BIRT into that IDE from using this BIRT update site:

https://download.eclipse.org/birt/updates/release/4.14.0
